chainer/functions/activation/log_softmax.py

```diff
@@ -108,15 +107,29 @@ def log_softmax(x, use_cudnn=True):
         ``log_softmax`` method is more stable.
 
     Args:
-        x (~chainer.Variable): Input variable.
-        use_cudnn (bool): If ``True``, cuDNN is enabled and cuDNN ver. 3 or
-            later is used, then this function uses cuDNN as the core
-            implementation.
+        X (:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`):
+            Input variable.
+            A :math:`n`-dimensional (:math:`n \\geq 2`) float array.
+        use_cudnn (bool): If ``True`` and cuDNN is enabled, then this function
+            uses cuDNN as the core implementation.
 
     Returns:
         ~chainer.Variable: Output variable.
+        A :math:`n`-dimensional (:math:`n \\geq 2`) float array, which is the
+        same shape with X.
 
     .. seealso:: :func:`~chainer.functions.softmax`
 
+    .. admonition:: Example
+
+        >>> x = np.array([[0, 1, 2], [0, 2, 4]], 'f')
+        >>> x
+        array([[ 0.,  1.,  2.],
+               [ 0.,  2.,  4.]], dtype=float32)
+        >>> F.log_softmax(x).data
+        array([[-2.40760589, -1.40760589, -0.40760589],
+               [-4.14293146, -2.14293146, -0.14293146]], dtype=float32)
```

Comment:
How about showing the result of `F.log(F.softmax(x))` also in this example? For example,
```
>>> F.log(F.softmax(x)).data
array([[-2.40760589, -1.40760589, -0.40760598],
       [-4.14293146, -2.1429317 , -0.1429316 ]], dtype=float32)
```
or
```
>>> np.allclose(F.log_softmax(x).data, F.log(F.softmax(x)).data)
True
```
etc.